### PR TITLE
Milestone 4 Regression Tests

### PR DESCRIPTION
**Pull request by gtrepta**
fixes: https://github.com/runtimeverification/iog-pm/issues/75

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -11,7 +11,8 @@ RUN    apt update        \
        llvm-11-dev       \
        libz3-dev         \
        libxml2-utils     \
-       gcovr
+       gcovr             \
+       npm
```
Comment (ehildenb) [inline Dockerfile:15]: Let's keep these sorted.

File: Jenkinsfile [hunk 2]
```diff
@@ -25,6 +26,28 @@ pipeline {
...
+            FAILING = "${env.WORKSPACE}/test/milestone4-contracts.failing"
+          }
+          steps {
+            dir('iog-pm') {
+              sh '''
+                git submodule init
+                git submodule update --depth 1
+                grep -v -x -F -f ${FAILING} ${CONTRACTLIST} > contracts.txt
+                export SOLC=${WORKSPACE}/build/solc/isolc
+                ./run-all-scripts.sh --filter contracts.txt --verbose --verbose
```
Comment (theo25) [inline Jenkinsfile:44]: Is the double `--verbose` flag needed here or just a typo?

**Review by theo25 — APPROVED** [review]
This looks good to me. I just left a minor comment/question.

File: Jenkinsfile (hunk 2 above)
Comment (gtrepta) [reply Jenkinsfile:44]: It's needed. The first option is used by `run-all-scripts.sh`, and the second one gets passed to `repos/run-solc.sh` which prints out warnings/errors from isolc. The harness is hacked together so it isn't elegant at this stage.

**Pull request merged**